tools/mkexport.sh

```diff
@@ -257,6 +257,9 @@ echo "HOSTLDFLAGS      = ${HOSTLDFLAGS}" >>"${EXPORTDIR}/scripts/Make.defs"
 echo "HOSTEXEEXT       = ${HOSTEXEEXT}" >>"${EXPORTDIR}/scripts/Make.defs"
 echo "LDNAME           = ${LDNAME}" >>"${EXPORTDIR}/scripts/Make.defs"
 echo "LDELFFLAGS       = ${LDELFFLAGS}" >>"${EXPORTDIR}/scripts/Make.defs"
+echo "NUTTX_ARCH       = ${NUTTX_ARCH}" >>"${EXPORTDIR}/scripts/Make.defs"
```

Comment:
why not export ARCH instead NUTTX_ARCCH?

Reply:
I want to avoid potential collision with external environnement and ARCH seems a bit too generic for this purpose. On top of that, I feel that it is is useful to have a clue of where the variable comes from (for isntance when looking in the CMake cache of a big project).

Reply:
Ok